pkgs/misc/vim-plugins/default.nix

```diff
@@ -1,449 +1,30 @@
 # TODO check that no license information gets lost
-{ callPackage, config, lib, stdenv
-, python, cmake, vim, vimUtils, ruby
-, which, fetchgit, llvmPackages, rustPlatform
-, xkb_switch, fzf, skim
-, python3, boost, icu, ncurses
-, ycmd, rake
-, pythonPackages, python3Packages
-, substituteAll
-, languagetool
-, Cocoa, CoreFoundation, CoreServices
-}:
+{ callPackage, config, lib, stdenv, vimUtils, vim, darwin, llvmPackages }:
 
 let
 
-  _skim = skim;
+inherit (vimUtils.override {inherit vim;}) buildVimPluginFrom2Nix;
```

Comment:
Do we care about indentation in this file?

Reply:
@Mic92 fixed the indentation